front_end/src/reducers/TileReducer.js

```diff
@@ -0,0 +1,49 @@
+import {
+  REQUEST_ADD_TILE,
+  RECEIVE_ADD_TILE,
+  REQUEST_TILES,
+  RECEIVE_TILES,
+  REQUEST_TILE,
+  RECEIVE_TILE
+} from 'actions/Tiles/TileActions';
+
+const initialState = {
+  rows: [],
+  details: [],
```

Comment:
I see this pattern in all the models. `rows` is for the list view of all of the instances of the model. `details` is for when we have one instance selected. Is that correct? Although now I am realizing details is an array so I am not sure.